napalm_logs/base.py

```diff
@@ -40,14 +40,6 @@
 
 class NapalmLogs:
     def __init__(self,
-                 address='0.0.0.0',
```

Comment:
The listener, publisher, and authenticator are vital hence their configuration must be trivial and straight forward. The rest are not mandatory for the app to work, hence they can get more complicated to document, configure etc.